### DISCUSSION TITLE
Organization notifications routing banner has broken URL

### DISCUSSION THREAD

**Discussion by KyleMit** [community/community#39434] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

If you join an organization with a email notification policy, you may see a banner that looks like this:

![image](https://user-images.githubusercontent.com/4307307/202739459-bbde50c4-0ca2-4c9a-b570-90a473324043.png)

> It looks like you are opted in to receive OrgName email notifications, but you have an ineligible email address configured in your notification routing settings. In order to receive email notifications for this organization, you'll have to update your organization routing settings for OrgName to use an email from one of the following domains:
>
> * example.com

The "View notification settings" button points to:

<https://github.com/settings/notifications#organization_routing>

However, the `#organization_routing` section is moved from that page and no longer contains that section / id

Instead, it's been moved to the "Custom Routing" button on the notifications page which takes you here:

<https://github.com/settings/notifications/custom_routing>

Would probably be good to include the new URL in the banner

**Comment by stevenh** [comment]
Thanks for this @KyleMit, been seeing this banner and clicked the original link which as you said doesn't provide the relevant options.
